To-do list/Models/Date+Extension.swift

```diff
@@ -9,9 +9,9 @@ import Foundation
 
 extension Date {
     
-    func formatDate (date: Date) -> String {
+    func formatDate() -> String {
         let formatter = DateFormatter()
```

Comment:
Creating dateFormatter is not a cheap operation.  It's better to cache/reuse it

https://developer.apple.com/library/archive/documentation/Cocoa/Conceptual/DataFormatting/Articles/dfDateFormatting10_4.html#//apple_ref/doc/uid/TP40002369-SW1

Reply:
done